PopPUNK/__main__.py

```diff
@@ -758,10 +832,18 @@ def assign_query(dbFuncs, ref_db, q_files, output, update_db, full_db, distances
             combined_seq, core_distMat, acc_distMat = update_distance_matrices(refList, ref_distMat,
                                                                 ordered_queryList, distMat, query_distMat)
             complete_distMat = translate_distMat(combined_seq, core_distMat, acc_distMat)
+            
+            if assign_lineage:
+                rank_list = sorted([int(x) for x in R.split(',')])
```

Comment:
Repeated

Reply:
Resolved in effb8909e329c8ac57e1bc9aea09be1088f7021d.